src/components/home-children/review-list-children/ReviewCard.js

```diff
@@ -0,0 +1,21 @@
+import { shortenText } from "../../../utils/utils";
+
+const ReviewCard = ({ review }) => {
+  const { review_img_url, review_body, title, designer, votes, comment_count } =
+    review;
+
+  console.log(review_img_url);
+  return (
+    <>
```

Comment:
Lots of fragments that arent needed. The div wraps everything, so dont need to fragment it up.